pkg/extensions/search/schema.graphql

```diff
@@ -550,6 +550,8 @@ type Query {
         image: String!,
         "Sets the parameters of the requested page"
         requestedPage: PageInput
+        "Search term for specific CVE by title/id"
+        searchedCVE: String!
```

Comment:
searchedCVE should not be mandatory. And if it is not present, all CVEs should be returned, just like current behavior on main.
```suggestion
        searchedCVE: String
```